qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/UserAccountSteps.java

```diff
@@ -32,6 +32,7 @@
 	private static final String CLERK = RandomStringUtils.randomAlphabetic(8);
 	private static final String NURSE = RandomStringUtils.randomAlphabetic(8);
 	private static final String DOCTOR = RandomStringUtils.randomAlphabetic(8);
+	private static final String LAB_ASSISTANT = RandomStringUtils.randomAlphanumeric(8);
```

Comment:
any reason for adding another user account ?

Reply:
lemi change that and use the available accounts

Reply:
resolved!!